marconi-chain-index/src/Marconi/ChainIndex/Indexers/MintBurn.hs

```diff
@@ -85,18 +90,27 @@ data TxMintEvent = TxMintEvent
   }
   deriving (Show, Eq, Ord)
 
+data MintAssetRedeemer = MintAssetRedeemer
+  { mintAssetRedeemerData :: !C.ScriptData
+  , mintAssetRedeemerHash :: !(C.Hash C.ScriptData)
+  }
+  deriving (Eq, Ord, Show, Generic, SQL.FromRow, SQL.ToRow)
+
 data MintAsset = MintAsset
   { mintAssetPolicyId :: !C.PolicyId
   , mintAssetAssetName :: !C.AssetName
   , mintAssetQuantity :: !C.Quantity
-  , mintAssetRedeemerIdx :: !Word64
-  , mintAssetRedeemerData :: !C.ScriptData
-  , mintAssetRedeemerHash :: !(C.Hash C.ScriptData)
+  , mintAssetIdx :: !Word64
```

Comment:
What is that field exactly? If there is no redeemer, then I wouldn've thought this would go with `MintAssetRedeemer`.

Reply:
It's the position of the asset in the list of assets. I don't think it's used anywhere. I kept it for legacy purposes but we can trash it.